shared/actions/teams/creators.js

```diff
@@ -34,6 +34,20 @@ function leaveTeam(teamname: string): Constants.LeaveTeam {
   return {payload: {teamname}, type: 'teams:leaveTeam'}
 }
 
+function addToTeam(
+  name: string,
+  email: string,
+  username: string,
+  role: 0 | 1 | 2 | 3 | 4,
```

Comment:
Perhaps pass around a more descriptive type, and then convert to integers when passing to the RPC. Although wouldn't there be a type in the RPC def file somewhere with the right constants?

Reply:
The typing for this does need to be rethought, in most places we're using a string enum but the RPC calls take these values. I think it stems from a difference in how we pass/get data to/from the RPCs about this